### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by Sentient07**
Features added, 
1) Changing password
2) Resetting password by sending an email
Edited the existing templates, making the code compliant with Django 1.6

### CODE REVIEW

File: pgd/settings.py [hunk 1]
```diff
@@ -128,6 +128,7 @@
     'django.middleware.common.CommonMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
```
Comment (Sentient07) [inline pgd/settings.py:131]: Added this because I got an error when I tried adding users from Admin(for testing the features)

File: pgd/urls.py [hunk 2]
```diff
@@ -17,8 +19,53 @@
 
     # Uncomment the next line to enable the admin:
     #(r'^admin/(.*)', admin.site.root),
```
Comment (Sentient07) [inline pgd/urls.py]: The admin.site.root gave me an error when I tried uncommented that line. I've added a line (new way ?) for making admin accessible

File: pgd_core/templates/registration/login.html [hunk 3]
```diff
@@ -32,6 +33,7 @@
   <input type="hidden" name="next" value="{{ next }}" />
 </form>
 
-<p>Forgot your Password?<strong><a style="color:black;"href="{% url auth_password_reset %}">...Reset It</a></strong></p>
-<p>Not Registered?<strong><a style="color:black;" href="{% url registration_register %}">...Register</a></strong></p>
+<p>Forgot your Password?<strong><a style="color:black;"href="{% url 'auth_password_reset' %}">...Reset It</a></strong></p>
+<p>Not Registered?<strong><a style="color:black;" href="{% url 'registration_register' %}">...Register</a></strong></p>
```
Comment (Sentient07) [inline pgd_core/templates/registration/login.html:37]: According to Django 1.6 upgrade, the urls must be enclosed in quotations

**Comment by Sentient07** [comment]
Also, when i tried adding the path to the existing templates, the path I gave didn't override the default Django admin's template. So i created a new file copying contents for some, and editing and copying for others

**Comment by mathuin** [comment]
I really like what you've done here!  Have you written any tests for the new functionality?

**Comment by Sentient07** [comment]
I'm hoping to push that by tonight, along with a few changes . :-)

**Comment by mathuin** [comment]
Woo hoo!  I'm around most of the day today, so I'm looking forward to trying it out. :-)

**Comment by mathuin** [comment]
Very nice!  Code continues to pass all tests.  Thanks!

**Comment by Sentient07** [comment]
_glad_ :)

**Pull request merged**